x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/review_rule_upgrade/review_rule_upgrade_route.ts

```diff
@@ -120,7 +123,12 @@ const calculateRuleInfos = (results: CalculateRuleDiffResult[]): RuleUpgradeInfo
       diff: {
         fields: pickBy<ThreeWayDiff<unknown>>(
           ruleDiff.fields,
-          (fieldDiff) => fieldDiff.has_update || fieldDiff.has_conflict
+          (fieldDiff) =>
+            fieldDiff.has_update ||
```

Comment:
It looks like now we return everything except AAA. I wonder if it makes sense to simplify to just:
```ts
(fieldDiff) => !fieldDiff.diff_outcome === ThreeWayDiffOutcome.StockValueNoUpdate
```